src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -941,6 +944,12 @@ private Button createNewEntryFromIdButton() {
         return newEntryFromIdButton;
     }
 
+    private CheckMenuItem createSidePaneCheckMenuItem(SidePaneContainerView container, ActionFactory factory, SidePaneType paneType, Command toggleCommand) {
+        CheckMenuItem checkMenuItem = factory.createCheckMenuItem(paneType.getToggleAction(), toggleCommand, (container.isPaneVisibleProperty(paneType).get()));
+        checkMenuItem.selectedProperty().bindBidirectional(container.isPaneVisibleProperty(paneType));
```

Comment:
This will probably lead to a runtime error 'bound values cannot be set' since javafx wants to set the selected value directly. So you may want to create a listener or create a binding manually.

Reply:
Nice catch, I'll update it right away